src/base/bittorrent/tracker.h

```diff
@@ -48,54 +52,58 @@ namespace BitTorrent
 {
     struct Peer
     {
-        QHostAddress ip;
         QByteArray peerId;
-        int port;
+        ushort claimedPort = 0;  // self-claimed by peer
+        bool isSeeder = false;
 
-        bool operator!=(const Peer &other) const;
-        bool operator==(const Peer &other) const;
-        QString uid() const;
-        lt::entry toEntry(bool noPeerId) const;
-    };
+        // caching precomputed values
+        lt::entry::string_type endpoint;
+        lt::entry::string_type ip;
```

Comment:
It can contain not only ip so I would call it "address".

Reply:
OK.